### ISSUE TITLE
Error building a static file with source maps enbled

### ISSUE THREAD

**Issue by scinos** [jspm/jspm-cli#2127]
When I try to build a static file with
```
./node_modules/.bin/jspm build example/main.js --format cjs
``` 

I get the error
```
err  Error: Unable to calculate canonical name to bundle file:///Users/sergio/src/personal/cangurovlogs/jspm17/1/npm:systemjs-plugin-babel@0.0.16/babel-helpers/classCallCheck.js. Ensure that this module sits within the baseURL or a wildcard path config.
    at getCanonicalNamePlain (/Users/sergio/src/personal/cangurovlogs/jspm17/1/node_modules/systemjs-builder/lib/utils.js:227:13)
    at getCanonicalName (/Users/sergio/src/personal/cangurovlogs/jspm17/1/node_modules/systemjs-builder/lib/utils.js:150:19)
    at /Users/sergio/src/personal/cangurovlogs/jspm17/1/node_modules/systemjs-builder/lib/rollup.js:289:25
    at Array.map (native)
    at /Users/sergio/src/personal/cangurovlogs/jspm17/1/node_modules/systemjs-builder/lib/rollup.js:288:49
```

But if I skip sourcemaps (`--skip-source-maps`), it works fine.

I created a small project with my configuration here: https://bitbucket.org/scinos/example-jspm17/src.

The relevant sections are:

```
//package.json
   //...
  "jspm": {
    "name": "example",
    "main": "main.js",
    "directories": {
      "baseURL": "client"
    },
    "dependencies": {},
    "devDependencies": {
      "plugin-babel": "npm:systemjs-plugin-babel@^0.0.16"
    }
  }
```

```
//client/jspm.config.js
  //...
  paths: {
    "npm:": "jspm_packages/npm/",
    "example/": "src/"
  },
  browserConfig: {
    "baseURL": "/"
  },
```

**Comment by guybedford** [comment]
This seems to have been some form of regression in Rollup. If you update to the latest patch that seems to work. I've included this bump in https://github.com/systemjs/builder/commit/29b3fc38e74305f94725c5ef7996272e2d3b9f0c, but it should also be possible to alter this in node_modules manually.

**Comment by guybedford** [comment]
Released in 0.17.0-beta.30.
